# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../alsa-lib/versionsort.patch.uclibc
# Copyright (C) 2016 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- alsa-lib-1.1.3/src/ucm/parser.c.vanilla	2016-12-28 16:31:49.161759274 +0000
+++ alsa-lib-1.1.3/src/ucm/parser.c	2016-12-28 16:33:07.429760618 +0000
@@ -1419,7 +1419,7 @@
 		"%s", env ? env : ALSA_USE_CASE_DIR);
 	filename[MAX_FILE-1] = '\0';
 
-#if defined(_GNU_SOURCE) && !defined(__NetBSD__) && !defined(__FreeBSD__) && !defined(__sun)
+#if defined(_GNU_SOURCE) && !defined(__NetBSD__) && !defined(__FreeBSD__) && !defined(__sun) && !defined(__UCLIBC__)
 #define SORTFUNC	versionsort
 #else
 #define SORTFUNC	alphasort
--- alsa-lib-1.1.3/src/conf.c.vanilla	2016-12-28 16:33:23.265760890 +0000
+++ alsa-lib-1.1.3/src/conf.c	2016-12-28 16:33:32.337761046 +0000
@@ -3715,7 +3715,7 @@
 			int n;
 
 #ifndef DOC_HIDDEN
-#if defined(_GNU_SOURCE) && !defined(__NetBSD__) && !defined(__FreeBSD__) && !defined(__sun)
+#if defined(_GNU_SOURCE) && !defined(__NetBSD__) && !defined(__FreeBSD__) && !defined(__sun) && !defined(__UCLIBC__)
 #define SORTFUNC	versionsort
 #else
 #define SORTFUNC	alphasort
